### PR TITLE
 Pricing Page Progress (geode fork)

### PR DESCRIPTION
**Pull request by kelihansen**

### CODE REVIEW

**Comment by ryw** [comment]
* we should reduce vertical spacing of the checkboxes - on my 13" MBP even the developer/enterprise divs don't fully fit on the screen
* To take this live ASAP, we'll need to modify the lists to be in line w/ current product:

Change "Developer" to "Hobby"
* 200+ Extensions
* Limited Stacks
* Up to 2 Databases
* 0.25 vCPU & 1 GB Memory
* 5 GB Storage
* Unlimited Reads/Writes

Production:
* 200+ Extensions
* All Stacks
* Unlimited Databases
* 0.25 to 32 vCPU
* 10 GB to 1 TB Memory
* Unlimited Reads/Writes
* High Availability
* Sidecars & Apps

Enterprise:
* Private Data Plane
* Network Isolation
* Dedicated Technical Support
* Optional Managed Data Plane
* Advanced Enterprise Security
* Bring Your Own Cloud

**Comment by DarrenBaldwin07** [comment]
> @kelihansen
> 
> * we should reduce vertical spacing of the checkboxes - on my 13" MBP even the developer/enterprise divs don't fully fit on the screen
> * To take this live ASAP, we'll need to modify the lists to be in line w/ our current product:
> 
> Change "Developer" to "Hobby"
> 
> * 200+ Extensions
> * Limited Stacks
> * Up to 2 Databases
> * 0.25 vCPU & 1 GB Memory
> * 5 GB Storage
> * Unlimited Reads/Writes
> 
> Production:
> 
> * 200+ Extensions
> * All Stacks
> * Unlimited Databases
> * 0.25 to 32 vCPU
> * 10 GB to 1 TB Memory
> * Unlimited Reads/Writes
> * High Availability
> * Sidecars & Apps
> 
> Enterprise:
> 
> * Private Data Plane
> * Network Isolation
> * Dedicated Technical Support
> * Optional Managed Data Plane
> * Advanced Enterprise Security
> * Bring Your Own Cloud

Good callouts @ryw - spacing is still WIP but we will make sure all the copy gets updated

**Comment by kelihansen** [comment]
@ryw I've updated the feature lists, thank you. It helps that they're all shorter than before! What would the ideal card height be from your perspective? Would you simply like the Hobby and Enterprise cards to end up less than say, 760px tall? (That's the vertical space I have available on my 13" screen after accounting for the site header and Chrome UI)

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "css.customData": [
+    ".vscode/tailwind.json"
+  ]
+}
```
Comment (kelihansen) [inline .vscode/settings.json:5]: Happy to remove this if you don't want it - it's based on [this](https://github.com/tailwindlabs/tailwindcss/discussions/5258) to get rid of the yellow squiggles I get under `@apply`

**Comment by ryw** [comment]
@DarrenBaldwin07 should this be closed?

**Pull request closed without merging**